brave/brave-core
# fix(wallet): Remove Duplicate Swap Tokens

## Description 

Fixes a bug where tokens were being duplicated in the `Select Token` modal list in `Swaps`


Resolves <https://github.com/brave/brave-browser/issues/38997>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Open the `Wallet` and navigate to the `Swap` screen
2. Open the `Select Token` modal.
3. There should be no duplicate tokens in the list.

Before:

https://github.com/brave/brave-browser/assets/40611140/ba5489cd-357a-4025-8335-b91698572af4

After:

https://github.com/brave/brave-core/assets/40611140/ab136b19-9fe6-4d34-9d3e-a0469751d0b7
